### PR TITLE
Update Deck GL v9

### PR DESCRIPTION
**Pull request by padawannn**
- Update to deck V9
- Remove dropping features functionality
- Adapt `useCartoLayerProps` to only return properties necessary to interact with the widget system

### CODE REVIEW

**Comment by aaranadev** [comment]
@padawannn, can you review all workspace dependencies?

I see `"@carto/react-api": "^3.0.0-alpha.3",` when the last version is `"@carto/react-api": "^3.0.0-alpha.5"` and the `^` doesn't work in alpha,

**Review by aaranadev — APPROVED** [review]

**Review by menusal — APPROVED** [review]
LGTM

**Comment by padawannn** [comment]
> @padawannn, can you review all workspace dependencies?
> 
> I see `"@carto/react-api": "^3.0.0-alpha.3",` when the last version is `"@carto/react-api": "^3.0.0-alpha.5"` and the `^` doesn't work in alpha,

Done, thx

File: tsconfig.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "allowSyntheticDefaultImports": true,
     "noEmit": true,
     "baseUrl": ".",
-    "compileOnSave": true,
+    // "compileOnSave": true,
```
[the remark this answers is not in the document]
Comment (padawannn) [reply tsconfig.json:14]: According to typescript validor compileOnSave is not a valid option.

Added the entry in the changelog, thx!

**Pull request merged**